oc_dms_mirror/dms_mirror.py

```diff
@@ -332,9 +332,9 @@ def _generate_component_config(self, component):
 
         _component_str = str(_component)
         if client_code:
-            _component_str = _component_str.replace("$component", component).replace("$client", f".{client_code}")
+            _component_str = _component_str.replace("$component", component).replace(".$client", f".{client_code}")
         else:
-            _component_str = _component_str.replace("$component", component)
+            _component_str = _component_str.replace("$component", component).replace(".$client", "")
```

Comment:
Are you sure about this? I mean, after that we will not have dot before client. Or I missed something?

Reply:
Got it